src/app/components/filter/filter.component.html

```diff
@@ -15,6 +15,11 @@ <h1><span class="material-icons">filter_list</span> Filters</h1>
                         <div class="clearfix"></div>
                     </div>
                     <mat-divider class="divider"></mat-divider>
+                    <button mat-raised-button type="button" (click)="navigateBeheerSkills()">Beheer Skills&nbsp;
```

Comment:
Please do not use any Dutch words.
Also not sure if this button needs to be shown like this. We don't want any user to easily find the admin environment for adding/removing skills.
On the line below you show an icon. Don't think that's relevant for this button.

Reply:
navigateBeheerSkills changed to navigateMaintainSkills
Button removed, password will be added in a future feature

Reply:
navigateBeheerSkills is removed anyway from the html file